src/main/lrsql/init/git_data.clj

```diff
@@ -0,0 +1,16 @@
+(ns lrsql.init.git-data
+  (:require [clojure.edn :as edn]
+            [clojure.string]
+            [clojure.java.shell :refer [sh]]))
+
+(defn read-version []
+  (try (edn/read-string (slurp "lrsql/config/git-details.edn"))
```

Comment:
This still won't work. You need to wrap that path in `clojure.java.io/resource` to access the classpath.